virtwho/runner.py

```diff
@@ -322,9 +322,9 @@ def send_number(self, rhsm_log):
         elif "virtwho.main DEBUG" in rhsm_log or "rhsm.connection DEBUG" in rhsm_log:
             if "satellite" in self.register_type:
                 if self.mode == "local":
-                    msg = r"Response: status=200, " r'request="PUT /rhsm/consumers'
+                    msg = r'Response: status=200, request="PUT /rhsm/consumers"'
                 else:
-                    msg = r"Response: status=200, " r'request="POST /rhsm/hypervisors'
+                    msg = r'Response: status=200, request="POST /rhsm/hypervisors"'
```

Comment:
nope, this should be
`r'Response: status=200, request="POST /rhsm/hypervisors' as our conversation`